www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+     complete and accurate registration information. You are responsible for
+keeping your account information, including your contact and payment
+information, up to date. Gratipay takes our Participants' privacy seriously. We
+will only use or share your information and Content as described in our
+[Privacy Policy](/about/policies/privacy), which is incorporated into these
+Terms.
+
+  1. You are responsible for maintaining the security of your account
+     information, including your password. You agree to notify Gratipay
+immediately of any actual or suspected loss, theft, or unauthorized use of your
+password. Gratipay shall not be liable for any loss or damage arising from your
+failure to keep your account secure.
+
+  1. You may only use the Service if you are 13 years old or older, and you
+     represent to us that you are 13 or older. If you are under the age of 18,
+your legal guardian must consent to your use and must agree to be bound by
+these Terms.
+
+1. The Gratipay Model and General Rules
+
+  1. The Service is a platform to enable Teams of Gratipay Participants to
+     receive payments to fund Open Work. Open Work means that the Team provides
+a clear path for any individual to voluntarily begin contributing to the Team's
+work and to be paid for their contribution. Some examples of Teams performing
+Open Work include an open source software project that pays contributors, or a
+hackerspace that pays individuals to teach classes or manage its operations.
+
+  1. The Participant who establishes a Team is its Owner. Teams may optionally
+     have other Members.
+
+  1. When you establish a Team on the Service to fund your Open Work, you agree
+     that payments will be used exclusively to support the Open Work described
+in your Team's profile.
+
+  1. When you make a payment to a Team, it is a payment to fund the Open Work
+     the Team provides. You understand and acknowledge that your payment does
+not entitle you personally to receive any goods or services from the Team,
+unless the Team explicitly offers those things to Participants.
+
+  1. By accepting a payment from a Participant, a Team forms an agreement with
+     that Participant to perform the Open Work described in its profile. The
+Service is a platform to facilitate this agreement. Gratipay is not a party to
+the agreement, and is not responsible to either the Team or the Participant if
+either party breaches the terms of the agreement (e.g. if the Team fails to
+provide the promised services).
+
+  1. Gratipay does not provide refunds. Teams are responsible for producing the
+     Open Work they offer. Gratipay does not hold funds on behalf of Teams, and
+does not warrant or guarantee that Teams will provide the Open Work they
+promise.
+
+1. Payroll Rules
+
+  1. Teams can use the Service's Payroll feature to automatically divide
+     incoming payments among Team Members, according to the allocation set by
+the Members.
```

Comment:
> set by the **Owner and/or** Members.

I'd like to leave the door open for different distribution algorithms that give the Owner more control, because there seems to be demand for that (e.g., https://github.com/tiliado/nuvolaplayer.project/issues/1#issuecomment-100695142).